.shippable.yml

```diff
@@ -43,8 +43,15 @@ build:
             S3_PATH="s3://zephyr-logs/${LOG_TYPE}/${REPO_FULL_NAME}/${BUILD_NUMBER}";
           fi;
       - >
-          if [ "$RUN_COMPLIANCE" = "1" -a "$IS_PULL_REQUEST" = "true" ]; then
-            export COMMIT_RANGE=origin/${PULL_REQUEST_BASE_BRANCH}..${COMMIT}
+          if [ "$MATRIX_BUILD" = "1" ]; then
+              export ARCH="-a x86 -a arc -a nios2 -a riscv32"
+          else
+              export ARCH="-a arm"
+          fi;
+
+          if [ "$MATRIX_BUILD" = "1" -a "$IS_PULL_REQUEST" = "true" ]; then
+            export ARCH="-a x86";
```

Comment:
do we need ARCH set for doing doc builds & check-compliance.py?  If not, move this out of here.